types/chrome/index.d.ts

```diff
@@ -10022,9 +10022,12 @@ declare namespace chrome {
             total: number;
         }
 
+        /** @deprecated Use {@link CpuTime} instead. */
+        type ProcessorUsage = CpuTime;
```

Comment:
A `type` is not equivalent to an `interface` as a `type` cannot be extended. That's why I suggested `interface Old extends New {}`.

Reply:
This is one of the rare cases where it's fine to silence the lint.

If you think it's fine, then sure, I guess